### DISCUSSION TITLE
Archive old devices when disconnected

### DISCUSSION THREAD

**Discussion by rfl64** [netbox-community/netbox#9052] [Ideas]
Would be interesting to mark removed old devices (switches, routers, firewalls, hosts, etc) and assign a new device that replaces the old one.
Connections (cables) and other related info could be keep if right.

Thanks

**Comment by candlerb** [comment]
Netbox isn't really an asset tracking system: it documents what you have right here and now, not the lifecycle of a specific asset (when it was purchased, its service history etc).

So the standard way to deal with this in Netbox is simply to update the serial number and/or asset tag and/or MAC addresses of the existing Netbox "Device" in-situ, such that the same device is now provided by a different asset.

In other words, a Netbox "Device" is really just a logical record that says "some asset is currently deployed with name N performing role R, at location Y with cables Z1 Z2 Z3 (etc)".  You can think of the serial number and/or asset tag as a foreign key into your asset management system.

**Reply by rfl64** [reply]
thanks for your fast reply. I understand, but i believe just tagging devices as disabled could give a brief history. Feeding database is big time consuming task and detete records once entered (i think) is waste time.

thank you very much.

**Comment by cb3inco** [comment]
I too was wondering about this in a similar vein in archiving a whole site. Perhaps I'm in a weird scenario where some of my projects and management stagnate for various reasons so it would be easier to archive them or hide them from view until I need to make them active again. 

It's a bit of a lot of work to delete it, and then re-add and re-document everything when I'd rather archive/disable.
